Os/test/ut/OsTestMain.cpp

```diff
@@ -49,7 +49,7 @@ TEST(Nominal, DISABLED_IntervalTimerTest) {
    intervalTimerTest();
 }
 TEST(Nominal, FileSystemTest) {
-   fileSystemTest();
+//   fileSystemTest();
```

Comment:
Uncomment code, or remove.

Reply:
Restored the UT for now.  Once Directory and FileSystem are refactored then this test will be no longer needed.